src/modules/dashboard/components/ColonyHomeActions/ColonyActionsDialog.css

```diff
@@ -0,0 +1,20 @@
+.content {
+  padding-bottom: var(--pad-huge);
+}
+
+.header {
+  position: relative;
+  padding: 0 40px;
+  padding-top: var(--pad-huge);
```

Comment:
And a top padding of `52px`, but you can set it to `60`, just so that we don't have that different of values:

![Screenshot from 2020-11-11 11-01-04](https://user-images.githubusercontent.com/1193222/98791226-6570d400-240d-11eb-9683-96043be8ae57.png)